src/main/java/seedu/address/logic/commands/ModifyCommand.java

```diff
@@ -54,14 +57,17 @@ public class ModifyCommand extends InternshipCommand {
             + PREFIX_APPLICATION_STATUS + "Yet to apply "
             + PREFIX_START_DATE + "20/01/2023 "
             + PREFIX_DURATION + "3 "
+            + PREFIX_DEADLINE + "25/12/2022 "
             + PREFIX_REQUIREMENT + "C++ "
             + PREFIX_REQUIREMENT + "Python";
 
     public static final String MESSAGE_EDIT_INTERNSHIP_SUCCESS = "Edited Internship: %s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_INTERNSHIP =
-            "This internship application already exists in the tracker.";
-
+            "This internship entry with the corresponding company name and role already exists in the application.";
+    public static final String MESSAGE_DEADLINE_CONSTRAINTS =
+            "Deadline should only contain numbers and slashes. It must follow the form DD/MM/YYYY. It must be earlier"
```

Comment:
Perhaps can standardise whether to use internship entry vs Internship.

Reply:
changed to internship entry's